glottolog3/views.py

```diff
@@ -428,6 +429,29 @@ def languages(request):
     res.update(map=map_, languoids=languoids)
     return res
 
+@view_config(
+    route_name='glottolog.add_languoid',
+    request_method='POST',
+    renderer='json')
+def add_languoid(request):
+    json_data = request.json_body
+
+    try:
+        languoid = LanguoidSchema().load(json_data)
+    except ValueError:
+        return {'error': 'Not a valid languoid level'}
+    if languoid.errors:
+        return {'error': languoid.errors}
+
+    try:
+        DBSession.add(languoid.data)
+        DBSession.flush()
+    except exc.SQLAlchemyError as e:
+        DBSession.rollback()
+        return {'message': 'ERROR: {}'.format(e)}
```

Comment:
should follow the same `{'error': ... }` format as the above errors. I know we're not being consistent with identifiers right now but using "error" as the key seems better to me.